dist/bin/bootstrap-environment

```diff
@@ -54,6 +54,16 @@ function setConfigVariables {
 }
 
 function setSecretsVariables {
+  if [[ -e $PERSIST_DIR/authelia/db.sqlite3 ]]; then
+    echo "${red}Authelia's database has already been populated. "
+    echo "Not generating new secrets.${normal}"
+
+    export GRIST_OIDC_IDP_CLIENT_ID
+    export GRIST_OIDC_IDP_CLIENT_SECRET
+    export DEX_CLIENT_SECRET
+    return
```

Comment:
Are these just re-exporting the values currently in the environment config? Wondering if there's any risk of these values being incorrectly overwritten when `envsubst` runs.

Reply:
Right, normally these are exported below, but in order to not over-write the existing values, we just re-export the ones we got from sourcing the existing env file in a previous step.